### PR TITLE
Add v1.1.0, v1.1.1 and v1.1.2 Release notes

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Add v1.1 Release notes.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@mheon PTAL.  Feel free to suggest/edit or if you want cherry pick, touchup and push a new variant.  Once this goes through, I'll do similar for v1.1.2 unless you beat me to it.

File: _posts/2019-02-26-podman-release-v1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+rootless containers and more.  All the details follow!
+
+<!--readmore-->
+
+## Changes
+
+### Features
+
+ * Added --latest and --all flags to podman mount and podman umount
+ * Rootless Podman can now forward ports into containers (using the same -p and -P flags as root Podman)
```
Comment (mheon) [inline _posts/2019-02-26-podman-release-v1.1.0.md:28]: Highlight this one - this was the big thing in 1.1, I think

File: _posts/2019-02-26-podman-release-v1.1.0.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+title: Podman v1.1.0 Released
+layout: default
+author: tsweeney
+categories: [releases]
+tags: community, open source, podman
+---
+
+# Podman Release 1.1.0
```
Comment (mheon) [inline _posts/2019-02-26-podman-release-v1.1.0.md:9]: I'd change this to 1.1.2 - we don't want anyone using 1.1.0, it's buggy

Comment (mheon) [reply _posts/2019-02-26-podman-release-v1.1.0.md:9]: (No real extra changes from the changelog to include, it was just bugfixes in 1.1.1 and 1.1.2)

Comment (TomSweeneyRedHat) [reply _posts/2019-02-26-podman-release-v1.1.0.md:9]: I was planning to add a separate RA for 1.1.2.  I'll work that up with this PR.  I think it's important to have a RA for each one we release, even if tiny.

File: _posts/2019-02-26-podman-release-v1.1.0.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply _posts/2019-02-26-podman-release-v1.1.0.md:28]: will do, thx!

File: _posts/2019-02-26-podman-release-v1.1.0.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+title: Podman v1.1.0 Released
```
Comment (TomSweeneyRedHat) [inline _posts/2019-02-26-podman-release-v1.1.0.md:2]: This should be v1.1.1

Comment (TomSweeneyRedHat) [reply _posts/2019-02-26-podman-release-v1.1.0.md:2]: nope, dyslexic moment, 1.1.0 is correct.

**Comment by TomSweeneyRedHat** [comment]
@mheon added v1.1.1 and v1.1.2 RA's and also touched up highlight in 1.1.0 as you suggested.  PTAL.

File: _posts/2019-03-05-podman-release-v1.1.2.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+tags: community, open source, podman
+---
+
+# Podman Release 1.1.2
+
+![podman logo](https://podman.io/images/podman.svg)
+
+## Podman has gone 1.1.2!
+
+After releasing Podman v1.1.1 a number of bug fixes
```
Comment (mheon) [inline _posts/2019-03-05-podman-release-v1.1.2.md:15]: I'd just say that we made many fixes to command line options and parsing

File: _posts/2019-03-01-podman-release-v1.1.1.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+tags: community, open source, podman
+---
+
+# Podman Release 1.1.1
+
+![podman logo](https://podman.io/images/podman.svg)
+
+## Podman has gone 1.1.1!
+
+After releasing Podman v1.1.0 a number of miscellaneous changes and several bug fixes
```
Comment (mheon) [inline _posts/2019-03-01-podman-release-v1.1.1.md:15]: Same as 1.1.2 - we focused on fixing regressions in CLI options and parsing

File: _posts/2019-03-05-podman-release-v1.1.2.md (hunk 4 above)
Comment (TomSweeneyRedHat) [reply _posts/2019-03-05-podman-release-v1.1.2.md:15]: Changed to 
```
After releasing Podman v1.1.1 a number of bug fixes
focusing on command line options and parsing were added. 
All the details follow!
```

File: _posts/2019-03-01-podman-release-v1.1.1.md (hunk 5 above)
Comment (TomSweeneyRedHat) [reply _posts/2019-03-01-podman-release-v1.1.1.md:15]: Changed to: 

```
After releasing Podman v1.1.0 a number of miscellaneous changes and several bug fixes
focusing on command line options and parsing were added.                             
All the details follow!
```

**Comment by mheon** [comment]
LGTM

**Pull request merged**